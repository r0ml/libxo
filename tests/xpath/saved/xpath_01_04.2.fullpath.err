main: input '#run --libxo @fullpath:slax,pretty'
main: input '#run'
main: input '?one/two[three == "4"]'
--- 0: 12
    000012: type 74 (C_PATH), str 0 [], contents 1, next 0, prev 0 
        000001: type 70 (C_ELEMENT), str 1 [one], contents 0, next 3, prev 0 
        000003: type 70 (C_ELEMENT), str 7 [two], contents 4, next 0, prev 1 
            000004: type 75 (C_PREDICATE), str 0 [], contents 6, next 0, prev 0 
                000006: type 14 (L_EQUALS), str 19 [==], contents 7, next 0, prev 0 
                    000007: type 74 (C_PATH), str 0 [], contents 5, next 8, prev 0 
                        000005: type 70 (C_ELEMENT), str 13 [three], contents 0, next 0, prev 0 
                    000008: type 54 (T_QUOTED), str 22 [4], contents 0, next 0, prev 7 
main: status: zero
main: input '+one'
main: status: track
main: input '+two'
main: status: track
main: input '$three 4'
main: key: 'three'='4'
main: status: full
main: input '=output yes'
main: field: 'output'='yes'
main: status: full
main: input '-two'
main: status: track
main: input '+two'
main: status: track
main: input '$three 5'
main: key: 'three'='5'
main: status: track
main: input '=output no'
main: field: 'output'='no'
main: status: track
main: input '-two'
main: status: track
main: input '+two'
main: status: track
main: input '$three 4'
main: key: 'three'='4'
main: status: full
main: input '+four'
main: status: full
main: input '=five 6'
main: field: 'five'='6'
main: status: full
main: input '-four'
main: status: full
main: input '-two'
main: status: track
main: input '-one'
main: status: track
