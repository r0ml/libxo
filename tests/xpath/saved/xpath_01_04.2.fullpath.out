/one/two[three='4']/output='yes'
/one/two[three='4']/four/five='6'
